android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -140,21 +140,18 @@ protected void triggerLayoutInflation() {
             Log.e(TAG, "triggerLayoutInflation " + e);
         }
 
-        mSwapButton.setOnClickListener(v -> {
-            assert mJsonRpcService != null;
-            LiveDataUtil.observeOnce(
-                    mWalletModel.getCryptoModel().getNetworkModel().mDefaultNetwork,
-                    defaultNetwork -> {
-                        LiveDataUtil.observeOnce(
-                                mWalletModel.getCryptoModel().mIsSwapEnabled, isSwapSupported -> {
-                                    SwapBottomSheetDialogFragment swapBottomSheetDialogFragment =
-                                            SwapBottomSheetDialogFragment.newInstance(
-                                                    isSwapSupported);
-                                    swapBottomSheetDialogFragment.setNetwork(defaultNetwork);
-                                    swapBottomSheetDialogFragment.show(getSupportFragmentManager(),
-                                            SwapBottomSheetDialogFragment.TAG_FRAGMENT);
-                                });
-                    });
+        mBuySendSwapButton.setOnClickListener(v -> {
+            NetworkInfo ethNetwork = null;
+            // Always show buy send swap with ETH
+            if (mWalletModel != null) {
+                ethNetwork = mWalletModel.getNetworkModel().getNetwork(
+                        BraveWalletConstants.MAINNET_CHAIN_ID);
+            }
+            SwapBottomSheetDialogFragment swapBottomSheetDialogFragment =
+                    SwapBottomSheetDialogFragment.newInstance();
+            swapBottomSheetDialogFragment.setNetwork(ethNetwork);
```

Comment:
I'm not sure I understood that part, why do we always set a network to ETH in that place? What if SOL network for example selected?

Reply:
This was discussed with @jamesmudgett and James proposed to use ETH as default network for BSS actions. The idea is to avoid working with local networks while adapting state independent BSS, and to slowly move towards web based BSS UI.
Slack Info thread: https://bravesoftware.slack.com/archives/C04DTJT4MDG/p1682611752098199

Reply:
ok, sounds good if it's what we want to do.